dcm4che-tool/dcm4che-tool-dcmqrscp/src/main/java/org/dcm4che3/tool/dcmqrscp/DcmQRSCP.java

```diff
@@ -796,9 +796,9 @@ public static void main(String[] args) {
     }
 
     private static void configureAcceptedCallingAETitles(ApplicationEntity ae, CommandLine cl) {
```

Comment:
Move
```java
static void configureAcceptedCallingAETitles(ApplicationEntity ae, CommandLine cl)
```
to `org.dcm4che3.tool.common.CLIUtils`, so it can be used by DcmQRSCP, IanSCP, MppsSCP and StoreSCP.

Reply:
Will do.